sample/src/test/java/co/nimblehq/sample/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package co.nimblehq.sample
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
+    @Test
+    fun addition_isCorrect() {
+        assertEquals(4, 2 + 2)
+    }
+}
```

Comment:
try fixing all of these missing eof line 🙇

Reply:
@sleepylee sorry, but what is eof line?
Can I fix it by reformatting code?

Reply:
you will need to add a blank line at the `End Of the File` 🙇  as you can see github review has a red icon to indicate this missing part.

Reply:
@sleepylee I got it, thank you 👍